### PR TITLE
fix progress-bar on chromium browsers

### PR DESCRIPTION
**Pull request by szaimen**
Close https://github.com/nextcloud/nextcloud-vue/issues/4199

| Browser | Before | After |
|---|---|---|
| Chromium | image | image |
| Firefox | image | image |

### CODE REVIEW

**Review by artonge — APPROVED** [review]

File: src/components/NcProgressBar/NcProgressBar.vue [hunk 1]
```diff
@@ -113,6 +113,7 @@ export default {
 	// Browser specific rules
 	&::-webkit-progress-bar {
 		height: var(--progress-bar-height);
+		background-color: var(--color-background-dark)
```
Comment (artonge) [inline src/components/NcProgressBar/NcProgressBar.vue:116]: Or

```suggestion
		background-color: transparent
```

Comment (artonge) [reply src/components/NcProgressBar/NcProgressBar.vue:116]: Also need the `;` at the end of the line.

Comment (szaimen) [reply src/components/NcProgressBar/NcProgressBar.vue]: done

**Review by nimishavijay — APPROVED** [review]
Nice!

**Review by jancborchardt — APPROVED** [review]

**Pull request merged**